### DISCUSSION TITLE
dify中是否有添加全局供应商的部分？

### DISCUSSION THREAD

**Discussion by 1linS** [langgenius/dify#6120] [General]
我想对于每个租户内置一些供应商，使其能够使用准备好的供应商模型，而不是每新建一个租户均要重新添加。或者说，dify源码中每新建一个租户，是怎样对该租户进行供应商及模型的配置的呢？

**Comment by k8scat** [comment]
你部署了cloud版？

**Reply by 1linS** [reply]
社区版的

**Reply by 1linS** [reply]
直接通过docker镜像拉取的社区版
